app/controllers/conference_registrations_controller.rb

```diff
@@ -6,7 +6,7 @@ class ConferenceRegistrationsController < ApplicationController
 
   def new
     @registration = Registration.new(conference_id: @conference.id)
-    authorize! :new, @registration
+    authorize! :new, @registration, message: "Sorry, you can not register for #{@conference.title}. Registration limit exceeded or the registration is not opened."
```

Comment:
The registration is not **open** (instead of opened) sounds better in this case.